libs/sdk-react-native/src/index.tsx

```diff
@@ -416,6 +420,7 @@ export type ReverseSwapPairInfo = {
     feesPercentage: number
     feesLockup: number
     feesClaim: number
+    feesTotalEstimated?: number
```

Comment:
Needs renaming to `totalEstimatedFees`

Reply:
Done in f02c462fb1406ad4f07a94a1a9dfb440e140bec1